components/public-api-server/main.go

```diff
@@ -22,6 +22,10 @@ func main() {
 		logger.WithError(err).Fatal("Failed to initialize public api server.")
 	}
 
+	if err := register(srv); err != nil {
+		logger.WithError(err).Fatal("Failed to register services.")
+	}
+
```

Comment:
Was this change necessary to set up the routing or just overlooked in earlier PRs?

Reply:
Overlooked from other PRs.